subprojects/bounded_context/user/infra/src/main/kotlin/club/staircrusher/user/infra/adapter/in/security/UserBoundedContextSecurityConfig.kt

```diff
@@ -5,7 +5,7 @@ import org.springframework.context.annotation.Configuration
 import org.springframework.security.config.annotation.web.builders.HttpSecurity
 import org.springframework.security.web.SecurityFilterChain
 
-@Configuration
+@Configuration(proxyBeanMethods = false)
```

Comment:
```suggestion
@Configuration
```
이거 다른 pr에서 완전 구현이 바뀌어서, diff 안 생기게 걍 롤백해주심 감사하겠습니당

Reply:
아하 넵